src/core/ad.jl

```diff
@@ -1,14 +1,18 @@
 ##############################
 # Global variables/constants #
 ##############################
-const ADBACKEND = Ref(:forward_diff)
+const ADBACKEND = Ref(:forwarddiff)
 setadbackend(backend_sym::Symbol) = setadbackend(Val(backend_sym))
 function setadbackend(::Val{:forward_diff})
+    @warn("`Turing.setadbackend(:forward_diff)` is deprecated. Please use `Turing.setadbackend(:forwarddiff)` to use `ForwardDiff`.")
```

Comment:
Can't we just use `@deprecate` (or `depwarn` if more fine tuning is needed)? And maybe move them to the bottom of the files, as it seems to be the convention, see e.g. https://github.com/JuliaLang/julia/blob/master/base/deprecated.jl?

Reply:
Sure.